etc/listproviders/event.upload.asset.options.properties

```diff
@@ -30,8 +30,8 @@ list.name=eventUploadAssetOptions
 # Attachments and catalogs upload options are for new and existing events.
 # Only one file can be uploaded for each of these options: the uploaded file replaces existing elements of the same type and flavor in the mediapackage.
 # EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CLASS_HANDOUT_NOTES={"id": "attachment_class_handout_notes", "type": "attachment", "flavorType": "attachment", "flavorSubType": "notes", "displayOrder":1}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_DFXP={"id":"catalog_captions_dfxp", "type": "catalog", "flavorType": "captions", "flavorSubType": "timedtext", "displayOrder":2}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_WEBVTT={"id":"attachment_captions_webvtt", "type": "attachment", "flavorType": "text", "flavorSubType": "webvtt", "displayOrder":3}
+# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_DFXP={"id":"catalog_captions_dfxp", "type": "attachment", "flavorType": "captions", "flavorSubType": "dfxp", "displayOrder":2}
```

Comment:
@karendolan Interestingly, this change introduces the following problem:

Reproduction:
1. Uncomment line 33
2. Add an event (add a dfxp catalog in the Add Event wizard)

This results in:

`2018-03-20 10:45:26,593 | WARN  | (IndexServiceImpl:1228) - Unable to process asset metadata [{"flavorType":"presenter","multiple":false,"displayOrder":13,"id":"track_presenter","type":"track","title":"EVENTS.EVENTS.NEW.SOURCE.UPLOAD.NON_SEGMENTABLE","flavorSubType":"source"},{"flavorType":"captions","displayOrder":2,"id":"catalog_captions_dfxp","type":"attachment","title":"EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_DFXP","flavorSubType":"dfxp"},{"flavorType":"presentation","multiple":false,"displayOrder":14,"id":"track_presentation","type":"track","title":"EVENTS.EVENTS.NEW.SOURCE.UPLOAD.SEGMENTABLE","flavorSubType":"source"},{"flavorType":"text","displayOrder":3,"id":"attachment_captions_webvtt","type":"attachment","title":"EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_WEBVTT","flavorSubType":"vtt"}]
`

I have verified that changed catalog_captions_dfxp to attachment_captions_dfxp fixes the problem... OMG!

@miesgre: Seems you need to fix this ;-)

@karendolan Do you know why this happens?

Reply:
The id prefix needs to match the type value. It’s a shortcut to sorting the form items on the server side.

Reply:
@staubesv Done!